modules/tinymce/Gruntfile.js

```diff
@@ -330,11 +331,21 @@ module.exports = function (grunt) {
     copy: {
       core: {
         options: {
-          process: function (content) {
-            return content.
+          process: function (content, src) {
+            let processed = content.
               replace('@@majorVersion@@', packageData.version.split('.')[0]).
               replace('@@minorVersion@@', packageData.version.split('.').slice(1).join('.')).
               replace('@@releaseDate@@', packageData.date);
+
+            if (src === '../../README.md') {
+              // Update the "## License" section content to just mention license.md file instead of GPL and license.md.
```

Comment:
This is a bit confusing so maybe just add another comment explaining why npm and github are using different license and maybe the TINY-13411 reference for future-proofing.